frontend/packages/shipwright-plugin/src/components/build-list/BuildTable.tsx

```diff
@@ -52,16 +60,51 @@ export const BuildHeader = () => {
       props: { className: columnClassNames[3] },
     },
     {
-      title: '',
+      title: t('shipwright-plugin~LastRun'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
       props: { className: columnClassNames[4] },
     },
+    {
+      title: t('shipwright-plugin~LastRunStatus'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[5] },
+    },
+    {
+      title: t('shipwright-plugin~LastRunTime'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[6] },
+    },
+    {
+      title: t('shipwright-plugin~LastRunDuration'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[7] },
+    },
+    {
+      title: '',
+      props: { className: columnClassNames[8] },
+    },
   ];
 };
 
-export const BuildRow: React.FC<RowFunctionArgs<Build>> = ({ obj: build }) => {
+const getLatestBuildRun = (buildRuns, build) => {
```

Comment:
Please add types here:

```suggestion
const getLatestBuildRun = (buildRuns: BuildRun, build: Build) => {
```

And please move this utils function to the existing utils function where we have `getLatestRun`.